### ISSUE TITLE
Question about the output of the model

### ISSUE THREAD

**Issue by zsw1456419654** [SteffenCzolbe/neuralizer#1]
Thank you for your excellent work on Neuroimage processing tasks! We have a question about the output of the model. As the paper described, the model can handle many tasks such as image segmentation, super resolution and so on. The result of image segmentation seem to be a mask, which only includes 0 and 1. However, the output of other tasks are grayscale image. After reading your code, we find the output is derived from a convolution layer. So we are wondering how the model can generate a clean output when it applied to image segmentation, which demands the output is a 0-1 mask.

We may not have understood your paper, and we would greatly appreciate it if you could explain it to us！

**Comment by adalca** [comment]
I will let @SteffenCzolbe comment on what he actually did, but conceptually the network output need not be in [0, 1] -- for example, it could be the logits of the segmentation (which you sigmoid to get the mask) or something similar!

**Comment by zsw1456419654** [comment]
Thanks for your reply, but that's where we are wondering. Can it be understood that this method trains the backbone of Neuralizer and during inference, the method adds a new function like sigmoid to get the mask or grayscale image due to the specific tasks. In other words, the method trains the network but employ different models when it is applied?
